packages/flutter/lib/src/material/material_button.dart

```diff
@@ -68,7 +68,7 @@ class MaterialButton extends StatelessWidget {
     this.focusElevation,
     this.hoverElevation,
     this.highlightElevation,
-    this.disabledElevation,
+    this.disabledElevation = 0.0,
```

Comment:
I would modify the assert on line 90 to say
```dart
       assert(disabledElevation != null || disabledElevation >= 0.0),
```

Since `RawMaterialButton` does not allow a null `disabledElevation` anyway.

Reply:
Yes, that sounds sensible to me.

Reply:
I agree. This would e a breaking change because the current constructor is const and doing this change requires to remove the const keyword!

Reply:
Here some context for the problem I managed to find: https://github.com/dart-lang/sdk/issues/36511